File: swift/1436-e-destination-city.swift

class Solution {
    func destCity(_ paths: [[String]]) -> String {
        var start: [String] = []
        var end: [String] = []

        for path in paths {
            start.append(path[0])
            end.append(path[1])
        }

        for e in end {
            if !start.contains(e) {
                return e
            }
        }

        return ""
    }
}

File: python/1436-e-destination-city.py
# https://leetcode.com/problems/destination-city/
class Solution:
    def destCity(self, paths: List[List[str]]) -> str:
        start = []
        end = []

        for path in paths:
            start.append(path[0])
            end.append(path[1])
        
        for e in end:
            if e not in start:
                return e